### DISCUSSION TITLE
[QUESTION] How to locate the disk for longhorn？

### DISCUSSION THREAD

**Discussion by btdan** [longhorn/longhorn#3947] [General]
Hello，after the installation of longhorn and configuration following the official document， the longhorn is now working.
However, I found longhorn employ the whole disk space that not used yet.
Can  I configure the longhorn to use the special disk space, such as the directory   /data , not the whole disk space.
Thanks very much.

**Comment by derekbit** [comment]
Yes, you can add and remove the disks. Please refer to https://longhorn.io/docs/1.2.4/volumes-and-nodes/multidisk/.
